.test-defs/PlantTest.yaml

```
@@ -16,6 +16,6 @@ spec:
       -ginkgo.v -ginkgo.progress
```

```suggestion
      --v -ginkgo.v -ginkgo.progress -ginkgo  -ginkgo.noColor
```

For better readability in our logs